### ISSUE TITLE
A teams section carousel in the home page.

### ISSUE THREAD

**Issue by Mrinal81** [Algo-Phantoms/Algo-Phantoms-Frontend#69]
I am GSSOC'21 Participant. I would like to make a slidebar that would give information about team members on a auto play loop. It would look good on the webpage.
I would like to work on this issue please assign this to me.

**Comment by srisatya1197** [comment]
I want to contribute to this issue. Please assign me this task as a part of GSSoC'21

**Comment by Shruti3004** [comment]
Yours @Mrinal81

**Comment by Mrinal81** [comment]
@Shruti3004  I made the pull request. Kindly review my pull request ( #137 )

**Comment by ravikr126** [comment]
are you completed @Mrinal81 ???

**Comment by Mrinal81** [comment]
Yes @ravikr126

**Comment by harikesh409** [comment]
I think on adding this we can remove the team.html page all together as it is an repetition.

**Comment by RahulBansal0502** [comment]
@harikesh409 is this issue completed as @ravikr126 assigned me an issue which will be completed only after this.

**Comment by Mrinal81** [comment]
@harikesh409  can I remove the team.html ??

**Comment by harikesh409** [comment]
> @harikesh409 can I remove the team.html ??

as of now don't remove the page. Why did you close the PR?

**Comment by Mrinal81** [comment]
By mistake

**Comment by Mrinal81** [comment]
I make pr #137

**Comment by harikesh409** [comment]
@Mrinal81 Okay not a problem we will review the new PR.
